.huskyrc

```diff
@@ -0,0 +1,6 @@
+{
+  "hooks": {
+    "pre-commit": "npm run lint",
+    "post-checkout": "npm ci"
```

Comment:
This runs every time you check out a branch? Is that necessary?

Reply:
No. Noise from some initial tests. Removing.